test/unit/utils/address_translation_test.mli

```diff
@@ -0,0 +1,7 @@
+open Bap.Std
+open Core_kernel
+
+
+val example_project: Project.t option ref
```

Comment:
Since you don't use the example project in the unit test, could you remove it here?